apps/gro/steps.js

```diff
@@ -133,7 +133,96 @@ module.exports = {
     template: 'email',
     fields: ['email-text'],
     continueOnEdit: true,
-    next: '/post'
+    next: '/country'
+  },
+  '/country': {
+    fields: [
+      'country'
+    ],
+    forks: [{
+      target: '/address-outside',
+      condition: function checkIfUK(req) {
+        return req.form.values.country !== 'United Kingdom';
+      }
```

Comment:
Function name checkIfUK should be checkIfNotUK as not describing what the function is checking. This would be better written as a condition object, checking if UK and next/target being swapped:

```js
forks: [{
  target: '/postcode'
  condition: {
    field: 'country',
    value: 'United Kingdom'
  }
}],
next: '/address-outside'
```